examples/devtools-recorder-cfe-jp/.env.example

```
@@ -0,0 +1,6 @@
+OPENAI_API_KEY=sk-...
+
+STAGEHAND_CACHE_DIR=.cache/stagehand-e2e
+E2E_SCREENSHOTS_DIR=screenshots
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The E2E_SCREENSHOTS_DIR key should go before the STAGEHAND_CACHE_DIR key

(UnorderedKey)